webtau-reactjs/src/report/OverallSummary.js

```diff
@@ -25,7 +25,7 @@ function OverallSummary({report, onSwitchToHttpCalls, onSwitchToSkippedHttpCalls
     return (
         <div className="overall-summary">
             <CallsTiming report={report} onSwitchToHttpCalls={onSwitchToHttpCalls}/>
-            <UrlCoverageSummary report={report} onSwitchToSkippedHttpCalls={onSwitchToSkippedHttpCalls}/>
+            <OperationCoverageSummary report={report} onSwitchToSkippedHttpCalls={onSwitchToSkippedHttpCalls}/>
```

Comment:
same as previous css comment

Reply:
Not sure this one is the same really.  UrlCoverageSummary presumably doesn't get used in a UI report?  Would be weird to have this be called UrlCoverageSummary when all the things it contains are "operation something".

Reply:
I mean the word `operation` is too generic. Maybe `OpenApi` prefix?
In UI `operation` could be button click and coverage will be percentage of buttons clicked.

Reply:
I see.  I misunderstood.  Will update.